projects/igniteui-angular/src/lib/grid/column-hiding.spec.ts

```diff
@@ -1213,3 +1236,19 @@ export class GridWithGroupColumnsComponent implements AfterViewInit {
     }
 
 }
+
+@Component({
```

Comment:
Following the test optimizations we recently introduced, we should avoid declaring grid components in spec files. You should either find a component that is already defined (in grid-base-components or grid-samples), or create your new one in one of those files. For this particular scenario, I believe you can use the BasicGridSearchComponent or the PagingComponent and execute the code in changeColumnVisibility in the test itself.